src/VolleyManagement.Backend/VolleyManagement.Services/PlayerService.cs

```diff
@@ -68,16 +68,27 @@ public IQueryable<Player> Get()
         /// Create a new player.
         /// </summary>
         /// <param name="playerToCreate">A Player to create.</param>
-        public void Create(Player playerToCreate)
+        public Player Create(Player playerToCreate)
```

Comment:
How UI going to create instance of player?

Reply:
Method Create in PlayRepository should also take 5 parameters? So in current situation is better to use  Introduce Parameter Object)